qemu/tests/fio_perf.py

```diff
@@ -136,7 +108,26 @@ def fio_thread():
         """
         run fio command in guest
         """
-        session.cmd_status(run_cmd, cmd_timeout)
+        # generate instance with fio
+        fio = generate_instance(params, vm, 'fio')
+        try:
+            fio.run(run_fio_options)
+        finally:
+            fio.clean()
+
+    def clean_tmp_files(session, os_type, guest_result_file, timeout):
+        """
+        Clean temporary test result files, fio tool packages and fio processes inside guest after testing
```

Comment:
@hellohellenmao Could you change it back, `clean temporary test result file inside guest`

Reply:
Updated. Sorry for forgetting this.